etcd/etcd-live-cluster-reconfiguration.md

```diff
@@ -1,236 +1,350 @@
 # etcd cluster runtime reconfiguration on CoreOS Container Linux
 
-This document describes the reconfiguration or recovery of an etcd cluster running on Container Linux, using a combination of `systemd` features and `etcdctl` commands.
+This document describes the reconfiguration and recovery of an etcd cluster running on Container Linux, using a combination of `systemd` features and `etcdctl` commands. The examples given in this document show the configuration for a three-node Container Linux cluster. The IP addresses used in the document are not real and replace them with the corresponding IPs. 
 
-## Change etcd cluster size
+## Configuring etcd using Container Linux Config
 
-When [a Container Linux Config][cl-configs] is used to configure an etcd member on a Container Linux node, it compiles a special `/etc/systemd/system/etcd-member.service.d/20-clct-etcd-member.conf` [drop-in unit file][drop-in]. That is, the Container Linux Config below:
+When a [Container Linux Config][cl-configs] is used for configuring an etcd member on a Container Linux node, it compiles a special `/etc/systemd/system/etcd-member.service.d/20-clct-etcd-member.conf` [drop-in unit file][drop-in]. For example:
 
 ```yaml container-linux-config
 etcd:
+  name: demo-etcd-1
+  listen_client_urls: https://10.240.0.1:2379,http://0.0.0.0:4001
   advertise_client_urls: http://<PEER_ADDRESS>:2379
+  listen_peer_urls:            http://0.0.0.0:2380
   initial_advertise_peer_urls: http://<PEER_ADDRESS>:2380
-  listen_client_urls: http://0.0.0.0:2379,http://0.0.0.0:4001
-  listen_peer_urls: http://0.0.0.0:2380
-  discovery: https://discovery.etcd.io/<token>
+  initial_cluster:             demo-etcd-1=https://0.0.0.1:2380,demo-etcd-2=https://0.0.0.2:2380,demo-etcd-3=https://0.0.0.3:2380
```

Comment:
0.0.0.{1,2,3,...} are broadcast addresses, would be clearer to use use 10.240.0.{1,2,3,4}.
```
initial_cluster: demo-etcd-1=http://10.240.0.1:2380,demo-etcd-2=http://10.240.0.2:2380,demo-etcd-3=http://10.240.0.3:2380
```